container/webui/docker-compose.yaml

```diff
@@ -109,9 +109,6 @@ services:
     environment:
       MODE: "webui"
       MOJO_LISTEN: "http://0.0.0.0:9526"
-    depends_on:
-      db:
-        condition: service_healthy
```

Comment:
So we're relying on `wait_for_db_creation()` in `run_openqa.sh` here? I suppose this is actually good enough to avoid the deadlock.

Reply:
Yes, for this container we have the webui mode, so this line will be executed https://github.com/os-autoinst/openQA/blob/5c4cb4a68d872f81f8111fc297d3b37a214b6eed/container/webui/run_openqa.sh#L27 therefore will wait for db creation